.github/workflows/build-wheels.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           # These are the platform build strings provided to
           # cibuildwheel, with wildcarding. See
           # https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip
-        python-build: ['cp37*64', 'cp39*64', 'cp310*64']
+        python-build: ['cp37*64', 'cp39*64', 'cp310*64', 'cp311*64']
```

Comment:
[question] I was originally going to suggest getting rid of one of the other entries, since the reason we didn't add 3.8 to the matrix was to avoid CI overload.  

But then I see that 3.7, 3.9 and 3.10 are all recent VFX reference platforms.

So then that begs the question, why do we need to officially support 3.11?  Is our policy "recent VFX Reference platforms, plus the latest"?

Do we still have a use case for 3.7, do you think?

Reply:
> "recent VFX Reference platforms, plus the latest"?

That was my thinking

> Do we still have a use case for 3.7, do you think?

Yeah, sadly a lot of people are still on 20/21.

Reply:
@feltech you are a genius:
```
Test project /Users/tomcowland/dev/OpenAssetIO/build
    Start 1: openassetio.internal.install
1/8 Test #1: openassetio.internal.install .....................   Passed    0.45 sec
    Start 2: openassetio-python-venv
2/8 Test #2: openassetio-python-venv ..........................   Passed   10.09 sec
    Start 3: openassetio.internal.core-cpp-test
3/8 Test #3: openassetio.internal.core-cpp-test ...............   Passed    0.14 sec
    Start 4: openassetio.internal.core-c-test
4/8 Test #4: openassetio.internal.core-c-test .................   Passed    0.13 sec
    Start 5: openassetio.internal.python-bridge-test
5/8 Test #5: openassetio.internal.python-bridge-test ..........   Passed    0.22 sec
    Start 6: openassetio.internal.pytest.cmodule
6/8 Test #6: openassetio.internal.pytest.cmodule ..............   Passed    0.49 sec
    Start 7: openassetio.internal.pytest.package
7/8 Test #7: openassetio.internal.pytest.package ..............   Passed    5.20 sec
    Start 8: openassetio.internal.pytest.resources-examples
8/8 Test #8: openassetio.internal.pytest.resources-examples ...   Passed    0.41 sec
```
I'll update the PR

Reply:
Do you mind if I rebase as fixups don't really work as some commits become redundant.

Reply:
No worries, rebase away.

Reply:
Sorry I messed up the threads - happy with the plan for versions?